test/integration/options.js

```diff
@@ -64,15 +64,17 @@ module.exports = function (createFn, setup, dismantle) {
           return done(err);
         }
 
-        erm.defaults({
+        const defaults = {
```

Comment:
⚠️ There is no longer a `defaults` method. You should define default options in the app and spread them in calls to `serve`.